pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/common/Felles.kt

```diff
@@ -137,4 +141,26 @@ object Felles {
             }
         }
     }
+
+    object NavnPaaBruker : ParagraphPhrase<LangBokmalNynorskEnglish>() {
```

Comment:
Hva med å løse dette med en operator i stedet?
```
fun Expression<Bruker>.fulltNavn(): Expression<String> =
        Expression.UnaryInvoke(
            value = this,
            operation = BrukerFulltNavn
        )

object BrukerFulltNavn: UnaryOperation<Bruker, String>(), StableHash by StableHash.of("UnaryOperation.BrukerFulltNavn") {
    override fun apply(input: Bruker): String = input.fulltNavn()
}
```

Reply:
Vi har allerede extension funksjonen `fun Bruker.fulltNavn()`

Reply:
For så vidt fristande, men ulempa der er at sia UnaryOperation er sealed, må den klassa i så fall liggje i templates-pakka. Og der ser det ut for meg som vi har veldig generiske metodar så langt. Samtidig så er jo namnet på brukar, og kanskje også andre ting som kjem frå felles, noko som ofte er nyttig i brev, så kanskje den trade-offen er verdt det lell.

Reply:
Det at den er `sealed` er nok bare fordi den aldri ble endret til `abstract` slik vi har gjort for `BinaryOperation`. Samtidig så ser jeg at den brukes i et `when`-uttrykk, så det må nok endres før vi kan bytte til `abstract`. Felles-klassen er så innebygd i brevbaker, at jeg tenker det ikke er noe problem å legge den inn under UnaryOperation.